packages/compiler-cli/src/ngtsc/typecheck/src/template_symbol_builder.ts

```diff
@@ -577,3 +594,7 @@ export class SymbolBuilder {
 function anyNodeFilter(n: ts.Node): n is ts.Node {
   return true;
 }
+
+function sourceSpanEqual(from: ParseSourceSpan, to: ParseSourceSpan) {
+  return from.start.offset === to.start.offset && from.end.offset === to.end.offset;
```

Comment:
```suggestion
function sourceSpanEqual(a: ParseSourceSpan, b: ParseSourceSpan) {
  return a.start.offset === b.start.offset && a.end.offset === b.end.offset;
```
nit: I don't think "from" and "to" feel right, as it makes this seem more like a range comparison. How about just a simple "a" and "b"